src/main/java/esa/sen2vm/utils/Sen2VMConstants.java

```diff
@@ -0,0 +1,58 @@
+package esa.sen2vm;
+
+public class Sen2VMConstants {
+
+    /**
+     * For bandInfo purpose
+     */
+    public static final int RESOLUTION_10M = 10;
+    public static final int RESOLUTION_20M = 20;
+    public static final int RESOLUTION_60M = 60;
+
+    /**
+     * For GIPP purpose
+     */
+    public static final String GIPP_VIEWDIR_NAME = ".*GIP_VIEDIR.*";
+    public static final String GIPP_BLINDP_NAME = ".*GIP_BLINDP.*";
+    public static final String GIPP_SPAMOD_NAME = ".*GIP_SPAMOD.*";
+    public static final int NB_GIPP_MAX_FILE = 13;
+
+    /**
+     * File extensions possible
+     */
+    public static final String xml_extention_small = ".xml";
+    public static final String xml_extention_big = ".XML";
+    public static final String dbl_extention_small = ".dbl";
+    public static final String dbl_extention_big = ".DBL";
+
+    /**
+     * Error management
+     */
+    public static final String ERROR_QUATERNION_NULL_GPS = "error.quaternion.null.gps";
+
+    /**
+     * Rugged Manager initialisation
+     */
+    // Pixel size for band (in meters)
+    public static final double PIXEL_HEIGHT_10 = 10.0;
```

Comment:
Duplicated with "RESOLUTION_10M"
Shall we keep both or can we keep only one type ? Int ?

Reply:
Fix in commit: https://github.com/sen2vm/sen2vm-core/commit/0a27773d43fd0c06d350b08da5b0805b51e6be4d